Graph/models.py

```diff
@@ -17,7 +19,10 @@ class NodeMissingError(ValueError):
     pass
 
 
-class Node:
+class Node:#(models.Model):
+    # seq = models.CharField(max_length=255, blank=True)
+    # paths = models.ManyToOneRel(Path)
+
```

Comment:
Eventually, this is the Node object HaploBlocker will use, but not yet.

Reply:
Ah, and I oversaw that alreay........... so ignore some questions of mine below.